swift/ThorVGError.swift

```diff
@@ -0,0 +1,7 @@
+/// Errors that can occur while working with ThorVG.
+public enum ThorVGError: Error {
```

Comment:
I think these are enumerations for only Lottie rendering.

How about renaming to `LottieError` or something else?